Catappult/appcoins-wallet-android
# [ESKL-335] Leaderboard reward improvement

**What does this PR do?**

   Adds check if the game supports ranking rewards (api call). Applied code style to existing code. Handles case if user haven't played any live games.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Play sandbox game on a game that does not have ranking rewards. You should see normal rankings and no "Last winners" or "next reward" timer. If you havent played any live game you should also not see "your ranking" container.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-335](https://aptoide.atlassian.net/browse/ESKL-335)

**Code Review Checklist**



[ESKL-335]: https://aptoide.atlassian.net/browse/ESKL-335?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
